index.html

```diff
@@ -21,6 +21,7 @@
 	<script>
 		if(!window.jQuery) {
```

Comment:
Assume that if we can't find jQuery, it means we're offline and also can't find the Dropbox script.